astropy/units/tests/test_format.py

```diff
@@ -183,6 +183,15 @@ def test_cds_log10_dimensionless():
     assert u.dex(u.dimensionless_unscaled).to_string(format="cds") == "[-]"
 
 
+def test_cds_angstrom_str():
+    # Regression test for a problem noticed in
+    # https://github.com/astropy/astropy/pull/17527#discussion_r1880555481
+    # that the string representation of the cds version of Angstrom was "AA".
+    assert str(u.cds.Angstrom) == str(u.Angstrom) == "Angstrom"
```

Comment:
Using multiple comparisons in a single assert statement is not a very good practice because if one of them fails then it can be more difficult to figure out which one it was. But if we deprecate and eventually remove `u.cds` then this test is only temporary.